src/classes/STG_InstallScript.cls

```diff
@@ -181,32 +184,6 @@ global without sharing class STG_InstallScript implements InstallHandler {
         }
     }
 
-    /*******************************************************************************************************
```

Comment:
The changes in this file, besides the new sendTelemetryData() method, are just a result of me re-arranging the methods to go in order of global -> public -> private